tuneme/migrations/0005_auto_20180926_1204.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2018-09-26 10:04
+from __future__ import unicode_literals
```

Comment:
Note: rename the migration files to describe what the migration was for

Reply:
:+1: